std/xml.d

```diff
@@ -2970,3 +2982,17 @@ private
         throw new XMLException(s);
     }
 }
+
+unittest
+{
+    string test_xml = `<?xml version="1.0" encoding='UTF-8'?><stream:stream xmlns:stream="http://etherx.'jabber'.org/streams" xmlns="jabber:'client'" from='jid.pl' id="587a5767" xml:lang="en" version="1.0"></stream:stream>`;
```

Comment:
The [style guide](https://dlang.org/dstyle.html) says: "[lines] can never exceed 120 characters." I'm not sure how serious we're being with that, though.